editors/code/tsconfig.json

```diff
@@ -1,22 +1,27 @@
 {
     "compilerOptions": {
         "module": "commonjs",
-        "target": "es2019",
+        //"target": "es2019",
+        "target": "es2020",
         "outDir": "out",
         "lib": [
-            "es2019"
+            //"ES2019"
+            "ES2020"
         ],
-        "esModuleInterop": true,
-        "allowSyntheticDefaultImports": true,
+        //"esModuleInterop": true,
+        //"allowSyntheticDefaultImports": true,
         "sourceMap": true,
         "rootDir": "src",
-        "strict": true,
-        "noUnusedLocals": true,
-        "noUnusedParameters": true,
-        "noImplicitReturns": true,
-        "noFallthroughCasesInSwitch": true,
-        "newLine": "LF"
+        //"strict": true,

```

Comment:
Why were these disabled?

Reply:
I think I was trying to figure out why I couldn't debug the project when I commented out those lines. It turns out I already had an instance of the running project open. So I closed that vscode window and the project ran as expected.
I usually have a few vscode instances open at once so it can be easy to miss something like that.